tests/test_other.py

```diff
@@ -10233,6 +10233,63 @@ def test_split_main_module(self):
     self.assertIn('Hello from main!', result)
     self.assertIn('Hello from lib!', result)
 
+  def test_split_module_ex(self):
+    initialTableSize = 5632
+    self.set_setting('SPLIT_MODULE')
+    self.set_setting('WASM', 1)
+    self.set_setting('WASM_BIGINT')
+    self.set_setting('EVAL_CTORS', 0)
```

Comment:
Remove this line. I doubt its doing anything because IIRC this setting doesn't work with wasm today.